content/support/changelog/school/index.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: layout-sidebar
+name: School (Beta)
+description: Information on additions and changes for the School API (Beta).
+order: 210
+published: true
+showInNav: false
+icon: fa fa-credit-card
```

Comment:
Change the icon here to `fa-graduation-cap`. I don't think anyone will see it, but better safe than sorry.